docs/source/model_doc/blenderbot.rst

```diff
@@ -0,0 +1,66 @@
+Blenderbot
+----------------------------------------------------
```

Comment:
We now have all the underlines at 119 characters long to make copy-pasting easier!

Reply:
Yes, you can easily copy-paste any line in any other rst file (for section titles too please).

Reply:
done, thx